waspc/src/Parser/Common.hs

```diff
@@ -136,6 +137,9 @@ waspNamedClosure name = do
       closureStart = L.symbol ("{=" ++ name)
       closureEnd = L.symbol (name ++ "=}")
 
+waspArray :: Parser a -> Parser [a]
```

Comment:
maybe add explanation/example? Also why array, maybe it should be list? that is kinda more general concept, array is more implementation stuff imo